docs/fundamentals/tools-and-productivity.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Tools and productivity in .NET
+author: IEvangelist
+description: Learn about the various tooling, software development kits, diagnostic tools, and code analyzers available to .NET developers.
+ms.author: dapine
+ms.date: 10/20/2020
+ms.topic: overview
+---
+
+# Tools and productivity in .NET
+
+In this article, you'll learn about the various tools available to .NET developers. With .NET, you have a robust software development kit (SDK), that includes a command-line interface (CLI) namely, the .NET CLI. The .NET CLI powers many of the features throughout the .NET-ready integrated development environments (IDEs). This article also provides resources to productivity capabilities, such as .NET CLI tools for diagnosing performance issues, memory leaks, high CPU, deadlocks, and as well as tooling support for code analysis.
```

Comment:
```suggestion
In this article, you'll learn about the various tools available to .NET developers. With .NET, you have a robust software development kit (SDK) that includes a command-line interface (CLI). The .NET CLI powers many of the features throughout the .NET-ready integrated development environments (IDEs). This article also provides resources to productivity capabilities, such as .NET CLI tools for diagnosing performance issues, memory leaks, high CPU, deadlocks, and tooling support for code analysis.
```

Reply:
"robust" SDK sounds like marketing language

"powers" is vague

"resources to productivity capabilities" sounds awkward

Given the whole article is an overview, it's not necessary to go over all the details in the intro paragraph.  Could replace this with something like "This article provides an overview of the development and diagnostic tools available for .NET."